Source/Atomic/Resource/Image.cpp

```diff
@@ -1211,6 +1216,91 @@ bool Image::SaveJPG(const String& fileName, int quality) const
         return false;
 }
 
+bool Image::SaveDDS(const String& fileName) const
+{

```

Comment:
Can leave it in on all platforms, but guard for windows and have other platforms return false and log a warning or error?